x-pack/platform/plugins/shared/features/server/oss_features.ts

```diff
@@ -830,7 +830,7 @@ const reportingFeatures: {
             ),
             includeIn: 'all',
             minimumLicense: 'gold',
-            savedObject: { all: [], read: [] },
+            savedObject: { all: ['scheduled_report'], read: [] },
```

Comment:
Update the existing reporting subfeature privileges to provide access to the `scheduled_report` saved object so it can be created when users schedule reports.